cpp/tests/utilities/type_lists.hpp

```diff
@@ -102,7 +103,17 @@ using FloatingPointTypes = cudf::test::Types<float, double>;
  * TYPED_TEST_CASE(MyTypedFixture, cudf::test::NumericTypes);
  * ```
  **/
-using NumericTypes = cudf::test::Types<int8_t, int16_t, int32_t, int64_t, float, double, bool>;
+using NumericTypes = cudf::test::Types<int8_t,
```

Comment:
I'd add the `uint`/`bool` types to `IntegralTypes` above, and then make `NumericTypes` be the `Concat` of `FloatingPointTypes, IntegralTypes`.